coverage/clover.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yup, coverage/ isn't ignore yet

Reply:
It is ignored. This PR has coverage files, as those files were added in before we talked about it.